Week2/Homework/assets/question-three.js

```diff
@@ -0,0 +1,9 @@
+let x;
+// x is not defined yet.
+console.log('the value of x will be: 5');
```

Comment:
Are you sure that the value here is 5? (you have not yet initialized the variable)

Reply:
Thank you Marciano. I misread the question.
The solution should be: 

let x;
// x is not defined yet.
console.log('the value of x will be: undefined');
console.log(x);

x = 6;
console.log('The value of x will be: 6');
console.log(x);

Reply:
ah good! yeah that one is correct:)